src/examples/create_datatype.cpp

```diff
@@ -63,11 +63,18 @@ int main(void) {
 
         // Create a more complex type with automatic alignment. For this the
         // type alignment is more complex.
-        CompoundType v_aligned({
-                                   {"u1", AtomicType<unsigned char>{}},
-                                   {"u2", AtomicType<short>{}},
-                                   {"u3", AtomicType<unsigned long long>{}}
-        });
+        // Use the constructor accepting a std::vector this time
+        std::vector<CompoundType::member_def> v_aligned_methods{
```

Comment:
I actually did just that 20 lines further up ;)

Reply:
didn't see it. I will revert this chunk